### PR TITLE
feat/admin-create-condominium

### LINKED ISSUE #3: [BE] Add condominium for ADMIN's
- [x] Create `POST` route 
- [x] Create `GET` post to get the `Syndicates` 
- [x] `Name`, `Address`, `Syndicate` 
- [x] Create swagger documentation

Cadastro_Condominio

### PR DESCRIPTION
**Pull request by RodrigoRVSN**
- [x] Create `POST` route 
- [x] Create `GET` post to get the `Syndicates` 
- [x] `Name`, `Address`, `Syndicate` 
- [x] Create swagger documentation

Cadastro_Condominio

### CODE REVIEW

File: src/app/user/dtos/CreateUserDto.ts [hunk 1]
```diff
@@ -12,6 +12,10 @@ export class CreateUserDto {
   @IsNotEmpty()
   password: string;
 
+  @ApiProperty()
+  @IsOptional()
+  phone: string;
```
Comment (gianlucabueno) [inline src/app/user/dtos/CreateUserDto.ts]: Is user phone optional in signup?

Comment (RodrigoRVSN) [reply src/app/user/dtos/CreateUserDto.ts]: this pr was a draft to add the condominium, the PR to update the phone field is here: https://github.com/RodrigoRVSN/set-api/pull/6

Comment (RodrigoRVSN) [reply src/app/user/dtos/CreateUserDto.ts]: But yes, i dont think we need to require the user to input the phone in sign up, makes more sense to keep optional for me

Comment (gianlucabueno) [reply src/app/user/dtos/CreateUserDto.ts]: Yes, makes sense for visitor, but for trustee and admin it's nescessary to have a phone number for communication between admin and trustee, in my vision

Comment (RodrigoRVSN) [reply src/app/user/dtos/CreateUserDto.ts]: i'll continue this in the update phone PR, i'll open this here

Comment (RodrigoRVSN) [reply src/app/user/dtos/CreateUserDto.ts]: done in [`00c3f29` (#6)](https://github.com/RodrigoRVSN/set-api/pull/6/commits/00c3f2986fccab847330f140ec898b24fdceb9e1) @gianlucabueno

**Review by gianlucabueno — APPROVED** [review]

**Review by mayoral-leonardo — APPROVED** [review]
All safe.

**Pull request merged**